serializer/test/test_serializer.c

```diff
@@ -135,14 +99,11 @@ void test_serialize_ta_find_transactions_by_tag(void) {
 int main(void) {
   UNITY_BEGIN();
 
-  RUN_TEST(test_deserialize_ta_get_tips);
```

Comment:
Would you append new test items accordingly?

Reply:
serializers from entangled has their own test cases there. And mock API tests remain same.